akka-docs-dev/rst/scala/http/routing-dsl/directives/form-field-directives/formFields.rst

```diff
@@ -3,8 +3,6 @@
 formFields
```

Comment:
cc @jrudolph @sirthias can you please have a look at this file, in respect to things mentioned in https://github.com/akka/akka/issues/17938 ? Did I miss something?